src/components/RecipeForm/style.css

```diff
@@ -4,6 +4,6 @@
     justify-content: space-between;
     height: calc(
         100vh - var(--header-height) - var(--recipe-select-height) -
-            var(--tab-height) - var(--footer-height)
+            var(--tab-height) - var(--footer-height) - var(--description-height)
```

Comment:
super nit: when switching from any recipe to multi-spheres, the `re-run` button shifts slightly (not fully fixed) because that recipe has more editable fields. I don’t think it’s blocking, just something I noticed.